app/src/main/res/values/strings.xml

```diff
@@ -424,4 +424,10 @@
     <string name="preference_header_about" translatable="false">@string/action_about</string>
     <string name="preference_header_contributing">Contributing</string>
     <string name="not_support_books">Books are not currently supported by Open Food Facts</string>
+    <string name="seconds_ago">seconds ago</string>
+    <string name="minutes_ago">minutes ago</string>
+    <string name="hours_ago">hours ago</string>
+    <string name="days_ago">days ago</string>
+    <string name="scan_first_string">Your viewed product history will be listed here.</string>
+    <string name="scan_first_string_continued">This history is for your eyes only and is stored locally.</string>
```

Comment:
This should be combined into one string, just stick a "\n" where there should be a line break.